### DISCUSSION TITLE
can not input Ctrl-\ on Mac German keyboard

### DISCUSSION THREAD

**Discussion by aivanise** [wezterm/wezterm#1890] [Q&A]
whenever I try to enter Ctrl-\ to send a SIGQUIT to whatever im running (on my keyboard this is Ctrl-RightOption-Shift-7) I get esc-Ctrl-Space instead of ctrl-\

> xxd
^[^_
00000000: 1b1f 0a

i have tried

keys = {
    -- make ctrl-\ work (or not)
    { key="phys:7", mods="CTRL|ALT|SHIFT", action="DisableDefaultAssignment" }
  }

in wezterm.conf, but nope.

does anyone have any idea? or how to get any other key combination to produce Ctrl-\?

**Comment by wez** [comment]
Did you try the latest nightly build?

**Reply by wez** [reply]
If it doesn't work for you on the nightly build, please file an issue using: https://github.com/wez/wezterm/issues/new?assignees=&labels=bug&template=bug.yml

**Comment by aivanise** [comment]
yup, the same in latest nightly, opened https://github.com/wez/wezterm/issues/1891
